apps/daffio/src/app/content/docs/api-docs/services/api-doc.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+
+import { ApiDocServiceProviderModule } from './api-doc-provider-service.module';
+
+@Injectable({
+  providedIn: ApiDocServiceProviderModule
+})
+export class DaffioDocService {
+
+  constructor(private http: HttpClient) {}
+
+  getDoc(path: string): Observable<any> {
+    return this.http.get('/assets/daffio/' + path + '.json');
```

Comment:
Move '/assets/daffio` into a class property.

Reply:
What about moving it into a CONST in the same file but outside of the class? Seems like it should be a const

Reply:
That would work.